manifests/prometheus/prometheus.yaml

```diff
@@ -43,6 +43,7 @@ spec:
     beta.kubernetes.io/os: linux
   replicas: 2
   resources:
+    limits: {}
```

Comment:
@jfchevrette i didn't verify, but is this fine?

Reply:
I've tested that empty map is valid and the end result is that no limits are set. This is the same as before.